source/analysis/AMAdditionAB.cpp

```diff
@@ -183,7 +182,7 @@ void AMAdditionAB::setInputDataSourcesImplementation(const QList<AMDataSource*>&
 	reviewState();
 
 	emitSizeChanged();
-	emitValuesChanged();
+	emitValuesChanged(AMnDIndex(rank(), AMnDIndex::DoInit));
```

Comment:
It looks like the arguments for emitValuesChanged() for this class have changed with this pull request. Does this mean that arguments have changed for all AB classes?